src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/BookingExtractionService.kt

```diff
@@ -60,12 +64,21 @@ class BookingExtractionService(
     val sentenceCalculation = sentence.sentenceCalculation
 
     if (sentence.releaseDateTypes.contains(SLED)) {
-      dates[SLED] = sentenceCalculation.expiryDate
+      if (sentence is DtoSingleTermSentence) {
+        dates[SLED] = sentenceCalculation.expiryDate.minusDays(sentenceCalculation.getAdjustmentBeforeSentence(AdjustmentType.REMAND).toLong())
```

Comment:
I think this logic should really be inside the `sentenceCalculation.expiryDate` function. The expiry date is referenced in a lot of places. (Same throughout this file.)

Reply:
Done.